<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVImporter</class>
 <widget class="QDialog" name="CSVImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Dataset</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Import Dataset</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileLoader">
     <item>
      <widget class="QLineEdit" name="filePath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resource.qrc">
         <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileImportTools">
     <item>
      <widget class="QLabel" name="fileStatus">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>No File Loaded</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="dataSeperatorSelectLabel">
       <property name="text">
        <string>Data Separator:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dataSeperatorSelect">
       <item>
        <property name="text">
         <string>Space or Tab (' ', '\t')</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Space (' ')</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tab ('\t')</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comma (',')</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customSeparatorInput">
       <property name="text">
        <string> </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="previewAssignmentTools">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget_6">
      <layout class="QVBoxLayout" name="previewTools">
       <item>
        <widget class="QLabel" name="previewTitle">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Preview of Contents:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="previewTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <layout class="QVBoxLayout" name="assignmentTools">
       <item>
        <widget class="QLabel" name="assignmentTitleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Columns Assigned for Required Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="assignmentListScrollWrapper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="assignmentListContainer">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>545</width>
            <height>83</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="assignmentList" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
